frontend/src/Editor/Components/Container.jsx

```diff
@@ -31,7 +33,7 @@ export const Container = function Container({
   return (
     <div
       data-disabled={disabledState}
-      className="jet-container"
+      className={`jet-container ${properties.loadingState && 'jet-container-loading'}`}
```

Comment:
Why do we need this additional class?

Reply:
class was added to make this center aligned  , will check why its not centered